DXRModules/DeusEx/Classes/DXRFlagsNGPMaxRando.uc

```diff
@@ -181,38 +183,45 @@ function NewGamePlus()
     p.saveCount=0;
     exp = 1;
     randomStart = settings.starting_map;
-    bingo_win = settings.bingo_win;
-    bingo_freespaces = settings.bingo_freespaces;
```

Comment:
Why did these get moved down?

Reply:
I was adding code to save newgameplus_curve_scalar over loops, so I placed it in the if(maxrando > 0) block. I noticed that some other values only relevant there were backed up outside the block. It was a bit confusing at first glance so I move everything there, within context and close to each other.

Reply:
Ok yeah this move is fine since the variables are only overwritten in SetDifficulty